### PR TITLE
update code_of_conduct.md(grammatical errors)

### PR DESCRIPTION
**Pull request by pra41**
Fixes #

**Screenshots**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -2,31 +2,31 @@
...
+* Public or private harassment.
+* Publishing others' private information, such as a physical or electronic address, without explicit permission.
+* Other conduct which could reasonably be considered inappropriate in a professional setting.
 
 ## Our Responsibilities
 
 Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
 
-Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+Project maintainers have the right and responsibility to remove, edit,or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive,or harmful.
```
Comment (abdulwd) [inline CODE_OF_CONDUCT.md]: This change should be reverted.

Comment (kelson42) [reply CODE_OF_CONDUCT.md]: Yes, after a comma, alwasy a space.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -2,31 +2,31 @@
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (iadeelzafar) [inline CODE_OF_CONDUCT.md]: ```our project and community```

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (abdulwd) [inline CODE_OF_CONDUCT.md]:
```suggestion
Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
```

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (abdulwd) [reply CODE_OF_CONDUCT.md]:
```suggestion
In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and community a harassment-free experience for everyone regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

**Comment by abdulwd** [comment]
@pra41 Can you commit the suggestions using the Github UI so that we can merge this? I'm not able to commit the suggestions as I do not have write access to your fork.

**Comment by kelson42** [comment]
Fixed in https://github.com/kiwix/kiwix-android/commit/89987cced9d2b76cbc492b62a2cf7fa8768d39c5

**Pull request closed without merging**